src/app/app.module.ts

```diff
@@ -15,7 +15,11 @@ import { TodoService } from './todo.service';
     AppComponent, ListComponent,EditComponent,AddComponent
   ],
   imports: [
-    BrowserModule,FormsModule
+    BrowserModule,FormsModule,RouterModule.forRoot([
+      {path:'',redirectTo:'todo',pathMatch:'full'},
```

Comment:
is todo route  defined  {path:'',redirectTo:'todo',pathMatch:'full'},?

Reply:
no sir! now solved it